File: LightningConstructs/Source/SpinGate/Declarations.vb

    Public Partial Class SpinGate(Of T)
        Private BaseReturnObject As T = Nothing
        Private BaseReturnFlag As System.Boolean = False
        Private BaseReturnCalledEarlyFlag As System.Boolean = False
        Private BaseRate As System.Double = 20.0
        Private BaseTimeoutCounter As System.Double = 0.0
        Private BaseTimeoutLimit As System.Double = 0.0
        Private ReadOnly SyncGovernor As Governor
        Private BaseWaitFrequencyMultiplyer As System.Double
    End Class

File: LightningConstructs/Source/SpinGate/Declarations.cs
namespace Lightning
{
    public partial class SpinGate<T>
    {
        private System.Object LockObject = new System.Object();
        private T BaseReturnObject = default!;
        private System.Boolean BaseReturnFlag = false;
        private System.Boolean BaseReturnCalledEarlyFlag = false;
        private System.Double BaseRate = 20.0;
        private System.Double BaseTimeoutCounter = 0.0;
        private System.Double BaseTimeoutLimit = 0.0;
        private Governor SyncGovernor = new Governor(20.0);
    }
}